File: implementation/scipamato/scipamato-common-wicket/src/test/java/ch/difty/scipamato/common/web/TestApplication.java
package ch.difty.scipamato.common.web;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;

@SpringBootApplication
public class TestApplication {

    public static void main(String[] args) throws Exception {
        new SpringApplicationBuilder().sources(TestApplication.class).run(args);
    }
}

File: implementation/scipamato/core/pubmed-api/src/adhocTest/kotlin/ch/difty/scipamato/core/TestApplication.kt
package ch.difty.scipamato.core

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.builder.SpringApplicationBuilder

@SpringBootApplication
open class TestApplication {

    @Suppress("SpreadOperator")
    fun main(args: Array<String>) {
        SpringApplicationBuilder()
            .sources(TestApplication::class.java)
            .run(*args)
    }
}
